attachment_s3/models/ir_attachment.py

```diff
@@ -46,17 +47,27 @@ def _get_s3_bucket(self, name=None):
 
         """
         host = os.environ.get('AWS_HOST')
+        port = os.environ.get('AWS_PORT')
```

Comment:
This shouldn't be needed as you can define the port on `AWS_HOST=http://minio1:9000`

Reply:
Are you sure you can define it that way? I just tried setting `AWS_HOST` to `http://minio:9000` and leaving `AWS_PORT` unset and I'm just getting a timeout. Even setting it to `minio:9000` doesn't seem to work.

If it was an endpoint URL I suppose it should work, but the word "host" implies that it's just a hostname and not an URL.

Reply:
Actually I might be wrong, because we use it in version 12.0 with `boto3`, and 11.0 wasn't migrated to `boto3` yet. 
I'd then suggest to consider the changes done in https://github.com/camptocamp/odoo-cloud-platform/pull/170 for 11.0 and probably rebase your PR on it.
There, we see URL is used as endpoint in the new version for connection with `boto3` (https://github.com/camptocamp/odoo-cloud-platform/pull/170/files#diff-5928316158bc6676a5a4498db5e517d9R60) and we actually use it in dev mode this way. (might make sense to rename the env var to `AWS_ENDPOINT` instead of `AWS_URL` but we might need to keep both for retro compatibility, @guewen what d'you think?) 

FWIW, this is the extension file we use for `docker-compose` (adapted from https://docs.min.io/docs/deploy-minio-on-docker-compose.html) :wink: :
```
# Extension file for the dev composition.

version: '2'
services:
  odoo:
    ports:
      - "8069:8069"
    depends_on:
      - minio1
    environment:
      - AWS_HOST=http://minio1:9000
      - AWS_ACCESS_KEY_ID=minio
      - AWS_SECRET_ACCESS_KEY=minio123
      - AWS_BUCKETNAME=customer_name-odoo-dev
  minio1:
    image: minio/minio:RELEASE.2019-02-20T22-44-29Z
    volumes:
      - data1:/data
    ports:
      - "9001:9000"
    environment:
      MINIO_ACCESS_KEY: minio
      MINIO_SECRET_KEY: minio123
    command: server http://minio1/data http://minio2/data http://minio3/data http://minio4/data
  minio2:
    image: minio/minio:RELEASE.2019-02-20T22-44-29Z
    volumes:
      - data2:/data
    ports:
      - "9002:9000"
    environment:
      MINIO_ACCESS_KEY: minio
      MINIO_SECRET_KEY: minio123
    command: server http://minio1/data http://minio2/data http://minio3/data http://minio4/data
  minio3:
    image: minio/minio:RELEASE.2019-02-20T22-44-29Z
    volumes:
      - data3:/data
    ports:
      - "9003:9000"
    environment:
      MINIO_ACCESS_KEY: minio
      MINIO_SECRET_KEY: minio123
    command: server http://minio1/data http://minio2/data http://minio3/data http://minio4/data
  minio4:
    image: minio/minio:RELEASE.2019-02-20T22-44-29Z
    volumes:
      - data4:/data
    ports:
      - "9004:9000"
    environment:
      MINIO_ACCESS_KEY: minio
      MINIO_SECRET_KEY: minio123
    command: server http://minio1/data http://minio2/data http://minio3/data http://minio4/data

## By default this config uses default local driver,
## For custom volumes replace with volume driver configuration.
volumes:
  data1:
  data2:
  data3:
  data4:
```